.github/workflows/azure-webapps-java-jar.yml

```diff
@@ -5,13 +5,10 @@ env:
   JAVA_VERSION: "17" #Java version to use
   DISTRIBUTION: zulu # Java distribution
   AZURE_WEBAPP_PACKAGE_PATH: ${{ github.workspace }} # path to your web app project
+  GH_TOKEN: ${{ github.token }}
```

Comment:
Is this being used anywhere?

Reply:
Yes, its needed to download artifacts using the GitHub cli